src/Components/ModalComponent/modalcomponent.css

```diff
@@ -0,0 +1,38 @@
+.modal-bg {
+    position: absolute;
+    z-index: 2;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
```

Comment:
Suggestion: You can use shorthand property for resetting the position.

Reply:
done used inset